docs/products/KPI-options.md

```diff
@@ -5,14 +5,14 @@ sidebar_label: KPI Options
 
 Key Performance Indicator (KPI) options are synthetic tokens that will pay out more rewards if a project’s KPI reaches predetermined targets before a given expiry date. Every KPI option holder has an incentive to improve that KPI because their option will be worth more. This is intended to align individual token holder interests with the collective interests of the protocol.
 
-Using UMA’s [Expiring Multi Party](/synthetic-tokens/what-are-synthetic-assets#the-expiringmultiparty-emp-contract-template) (EMP) contract template and [Optimistic Oracle](/getting-started/oracle.md), any project can create their own KPI Option tokens. These can be backed by any approved ERC-20 token and can be valued against any KPI that a project wants to improve! 
+Using UMA’s [Long Short Pair](/synthetic-tokens/long-short-pair) (LSP) contract template and [Optimistic Oracle](/getting-started/oracle.md), any project can create their own KPI Option tokens. These can be backed by any approved ERC-20 token and can be valued against any KPI that a project wants to improve! 
 
 ## Getting Started
 
 The process to launch a KPI options contract on UMA is surprisingly simple. No on-chain price feed or smart contract development is ever required. This is the typical process that you will need to follow to launch your own.
 
 1. You should submit two [UMA Improvement Proposals](/uma-tokenholders/umips) (UMIPs). The first should add your governance token as a supported collateral type. The second should define a methodology for how the UMA Optimistic Oracle should price your KPI options.
-2. Once these proposals are approved through UMA governance, you can launch your expiring KPI Options contract! This can be done in a few minutes by following the [EMP deployment tutorial](/developers/emp-deployment).
+2. Once these proposals are approved through UMA governance, you can launch your expiring KPI Options contract! This can be done in a few minutes by following the [LSP deployment tutorial](https://github.com/UMAprotocol/launch-lsp).
```

Comment:
I could possibly add a launch LSP tutorial using the one I created for the Polygon deployment